src/gui/MItem_menus.cpp

```diff
@@ -403,16 +404,26 @@ MI_LOAD_SETTINGS::MI_LOAD_SETTINGS()
 }
 
 void MI_LOAD_SETTINGS::click(IWindowMenu & /*window_menu*/) {
-    // FIXME: Some error handling/reporting
-    // TODO: Loading other things than just network
-    if (netdev_load_ini_to_eeprom()) {
+    bool network_settings_loaded = netdev_load_ini_to_eeprom();
+
+    if (network_settings_loaded) {
         notify_reconfigure();
     }
-
-// FIXME: Error handling
+    auto build_message = [](StringBuilder &msg_builder, string_view_utf8 name, bool ok) {
+        msg_builder.append_string_view(name);
+        msg_builder.append_string(": ");
+        msg_builder.append_string_view(ok ? _("Ok") : _("Failed"));
+        msg_builder.append_string("\n");
```

Comment:
Nitpick: You could use more efficient append_char here.

Reply:
Done.